ucgenerator/main.go

```diff
@@ -363,3 +364,14 @@ func isTypeSet(o *object) bool {
 	}
 	return false
 }
+
+// toTitleRegex finds words with a leading lowercase letter,
+// ignores dotted and underscored literals
+var toTitleRegex = regexp.MustCompile(`^[a-z]\w+[\s,$]`)
```

Comment:
We could've avoided the usage of `regexp` here, and all the checks, and instead do:

```go
strings.ToUpper(string(some_string[0]))[0]
```

WDYT?

Reply:
It ignores literals that look like a code snippet (that refer to a property field). Those literals have dots and underscores within, which normal words do not have.